miso-web/src/main/webapp/pages/editSample.jsp

```diff
@@ -388,7 +388,7 @@
           </td>
         </tr>
         <tr>
-          <td class="h">Concentration (nM):</td>
+          <td class="h">Concentration (${sampleConcentrationUnits}):</td>
```

Comment:
Could use `Constants.sampleConcentrationUnits` here rather than adding as a model attribute for this page, if you wanted.

Reply:
Not in this context. :-(

Reply:
Could, set it in JS on document load, but I think it's cleaner as a model attribute